### DISCUSSION TITLE
error: 'components_515c5644' has already been declared

### DISCUSSION THREAD

**Discussion by Iachevskiy** [nuxt/nuxt#18308] [Questions] [answered]
WARN  [SSR] Error transforming virtual: ...nuxt/plugins/server.mjs: Error: Parse failure: Identifier 'components_515c5644' has already been declared (7:7) 
Contents of line 7: import components_515c5644 from '/@id/__x00__virtual: ...nuxt/components.mjs'

components.mjs:
    declare module 'vue' {
      export interface GlobalComponents {
        'Header': typeof import('../components/Header.vue')['default'],
        'Nav': typeof import('../components/Nav.vue')['default'],
        'Breadcrumbs': typeof import('../components/breadcrumbs.vue')['default'],
        'ModalBackground': typeof import('../components/modal/background.vue')['default'],
        'ModalWindow': typeof import('../components/modal/Window.vue')['default'],
        'ProductList': typeof import('../components/product/list.vue')['default'],
        'ModalPageReview': typeof import('../components/modal/page/review.vue')['default'],
        'UxSvgIcon': typeof import('../components/ux/svg/icon.vue')['default'],
        'UxSvgSprite': typeof import('../components/ux/svg/sprite.vue')['default']
      }
    }
    export {}

what could be the problem?

**Comment by danielroe** [comment, marked as answer]
Would you open an issue?

**Reply by Iachevskiy** [reply]
I don't understand what you mean. What do I need to do?

**Reply by danielroe** [reply]
https://github.com/nuxt/framework/issues/new?assignees=&labels=bug%2Cnuxt3&template=bug-report-nuxt3.yml

**Reply by Iachevskiy** [reply]
well, thank you
